src/modules/bulk-minter/components/change/ChangeDonationField.tsx

```diff
@@ -0,0 +1,280 @@
+import { StyledCreatorsRow } from "../RoyaltiesCreators";
+//@ts-ignore
+import FeatherIcon from 'feather-icons-react';
+import creatorStandinImg from '../../../../assets/images/creator-standin.png';
+import { Debounce } from "src/utils/debounce";
+import styled from 'styled-components';
+
+const DEBOUNCE_INTERVAL_MS = 300;
+
+interface Props {
+    showDonationField: boolean;
+    creators: Creator[];
+    updateDonationFieldState: (show: React.SetStateAction<boolean>)=>void;
+    updateCreatorState: (value: React.SetStateAction<Creator[]>)=>void;
+    updateShowErrorState: (value: React.SetStateAction<boolean>)=>void;
+}
+
+const StyledSelectButton = styled(Paragraph)`
+  margin-right: 5px;
+  font-size: 14px;
+  cursor: pointer;
+  &:hover {
+    opacity: 0.8;
+  }
+`;
+
+const AlreadyAddedNotice = styled(Paragraph)`
+  margin-right: 5px;
+  font-size: 14px;
+`;
+
+const CharityRow = ({
+    charity,
+    onSelect,
+    enableSelectButton,
+    selectButtonText
+  }: {
+    charity: ChangeNonprofit;
+    onSelect: () => void;
+    enableSelectButton: boolean;
+    selectButtonText: 'Select' | 'Already Added ✅';
+  }) => {
+    const {name, icon_url, crypto, description, ein} = charity;
+    const {solana_address} = crypto;
+    return (
+      <div style={{ width: '100%' }}>
+      <StyledCreatorsRow className="minter-charity-row">
+        {icon_url ? (
+          <img height={32} width={32} className="rounded-full" src={icon_url} alt={name + '-logo'} />
+        ) : (
+          <img height={32} width={32} className="rounded-full"  src={creatorStandinImg} alt="generic-logo" />
+        )} 
+        <Paragraph
+          style={{
+            margin: '0 14px 0 6px',
+            maxWidth: 200,
+            fontSize: 14,
+            lineHeight: 1.2,
+          }}
+        >
+          {charity.name}
+        </Paragraph>
+        <FeatherIcon
+            className="creator-row-icon"
+            icon="external-link"
+            onClick={() => {window.open(CHANGE_BASE_URL + solana_address, "_blank")}}
+          />
+        <span style={{ marginLeft: 'auto' }}></span>
+        { enableSelectButton && <StyledSelectButton
+          onClick={onSelect}
+        >
+          {selectButtonText}
+        </StyledSelectButton> }
+        { !enableSelectButton && <AlreadyAddedNotice
+        >
+          {selectButtonText}
+        </AlreadyAddedNotice> }
+      </StyledCreatorsRow>  
+        <Paragraph
+          style={{ fontSize: 10, marginLeft:14, lineClamp: 4, textOverflow: 'ellipsis', overflow: 'hidden' }}
+        >
+          {description}
+        </Paragraph>
+        <Paragraph
+          style={{ fontSize: 10, marginLeft:14 }}
+          >
+          EIN: {ein}
+        </Paragraph>
+        </div>
+    );
+  };
+
+
+export const ChangeDonationField: FunctionComponent<Props> = ({
+    showDonationField,
+    creators,
+    updateDonationFieldState,
+    updateCreatorState,
+    updateShowErrorState
+  }: Props) => {
```

Comment:
```suggestion
export function ChangeDonationField(props: Props): JSX.Element {
```

and access props by `props.`